bindings/python/pymongoarrow/lib.pyx

```diff
@@ -56,6 +56,55 @@ cdef const bson_t* bson_reader_read_safe(bson_reader_t* stream_reader) except? N
         raise InvalidBSON("Could not read BSON document stream")
     return doc
 
+cdef str _get_human_readable_bson_type_t(bson_type_t type_t):
```

Comment:
`_get_human_readable_bson_type_t` is a bit of a mouthful. What about "_bson_type_name"?